dist/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>TEST</title>
+    </head>
+    <body>
+        <div class="center"></div>
```

Comment:
Feel free to use `<div id="root></div>`. This is a React convention and its okay to use id here.